testing/helm/run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-options="--set global.image.version=latest"
+options="--set global.image.version=someversion --set global.image.pullPolicy=Always"
```

Comment:
what is the significance of this change?
Did kubeval/kube-score complain re use of "latest" ? and IfNotPresent default?

Reply:
Yes

Reply:
ok